android/src/main/java/com/fidelreactlibrary/adapters/abstraction/VerificationSetupAdapter.java

```diff
@@ -0,0 +1,9 @@
+package com.fidelreactlibrary.adapters.abstraction;
+
+import com.fidelapi.entities.CardVerificationConfiguration;
+import com.facebook.react.bridge.ReadableMap;
+
+public interface VerificationSetupAdapter {
+
+    CardVerificationConfiguration map(ReadableMap data);
```

Comment:
I would suggest the name `adapt`, so that it doesn't read like you are mapping a map.